### ISSUE TITLE
Feature: Add utility function to read server-side resolver from another resolver

### ISSUE THREAD

**Issue by brandonroberts** [analogjs/analog#1055]
The `load` resolver can be called from within another resolver by accessing it through the route config.

```ts
export const routeMeta: RouteMeta = {
  resolve: {
    data: async(route) => {
      // call server load resolver for this route from another resolver
      const data = await route.routeConfig?.resolve?.['load']?.(route);

      return { ...data };
    }
  }
}
```

We can add a utility function for this named `getLoadResolver` and use it as `getLoadResolver(route)`.

```ts
import { getLoadResolver } from '@analogjs/router';

export const routeMeta: RouteMeta = {
  resolve: {
    data: async(route) => {
      // call server load resolver for this route from another resolver
      const data = await getLoadResolver(route);

      return { ...data };
    }
  }
}
```

_Originally posted by @brandonroberts in https://github.com/analogjs/analog/issues/1039#issuecomment-2069965764_

**Comment by Tenessy** [comment]
Hello @brandonroberts, can you assigned me to this task please ?

**Comment by brandonroberts** [comment]
@Tenessy yep, thanks!

**Comment by brandonroberts** [comment]
Closed by https://github.com/analogjs/analog/pull/1144  and released in `1.5.0`
